edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -163,6 +166,7 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                         string managementUri = configuration.GetValue<string>(Constants.EdgeletManagementUriVariableName);
                         string workloadUri = configuration.GetValue<string>(Constants.EdgeletWorkloadUriVariableName);
                         iothubHostname = configuration.GetValue<string>(Constants.IotHubHostnameVariableName);
+                        builder.RegisterInstance(new NullMetricsProvider() as IMetricsProvider);
```

Comment:
@lfitchett Is this something I should remove?

Reply:
Do we still want to expose edgehub and edgeAgent metrics on k8s? stuff like number of messages sent to deployment time? If so you'll need to replace the null metrics provider with a real one

Reply:
The real metrics are installed later - removed these null metrics.  Also, the metrics in edgeAgent - The edgeAgent_metadata host_information is not useful, but the rest of the data looks OK.